### DISCUSSION TITLE
Importing & exporting data

### DISCUSSION THREAD

**Discussion by PerpSearch** [benjaminjonard/koillection#1239] [Ideas]
I know this has come up before as I thought some sort of collection data import/export would be useful, but I been working with another person new to Koillection in order to help them organize things.   It occurred to me that it would also be useful to export or copy an existing choice list or template to use in another collection database.  

Thanks again for all you've done here.

**Comment by Lyca-x** [comment]
Bumping discussion thread.
When Exporting my collection, I only receive the path of "Collection"->"Item Name". Any information I have appended to the actual item via a template is not exported into the .csv. As a specific example, I have the collection "Games", and the item "Mario Kart 64" in it. The .csv only gives me these two values, instead of my added data fields like "Release Date", "Condition", "Play Status".

This functionality would allow me to mostly renege my currently existing, hardly maintainable, spreadsheet file to a pure "Make calculations with the data" instead of misusing it as a database as well.
